internal/pkg/gateway/endorsement.go

```diff
@@ -114,6 +115,17 @@ func (p *plan) processEndorsement(endorser *endorser, response *peer.ProposalRes
 	} else {
 		if !bytes.Equal(p.responsePayload, response.GetPayload()) {
 			logger.Warnw("ProposalResponsePayloads do not match (base64)", "payload1", b64.StdEncoding.EncodeToString(p.responsePayload), "payload2", b64.StdEncoding.EncodeToString(response.GetPayload()))
+			if logger.IsEnabledFor(zapcore.DebugLevel) {
```

Comment:
In the Warnw message above, users will receive the base64 difference but will typically want to know the actual difference. In the Warnw message, could you advise the user to enable "debug" for "gateway" component to get the actual differences?

Reply:
Ok, I'll do that.  Since the details will be logged by default, I'll remove the base64 encoded responses from the log.  These are pretty useless from a usability standpoint anyway.